test/attr.test.js

```diff
@@ -11,6 +11,16 @@ class Fixture extends React.Component {
 const it = createTest(<Fixture />)
 
 describe('#attr', () => {
+  it('fails when actual is not a enzymeWrapper', () => {
+    expect(() => {
+      expect(undefined).to.have.attr('key', 'somekey')
+      expect({ 'foo': 'bar' }).to.have.attr('key', 'somekey')
```

Comment:
I think all expectations starting from this one will never reached, because the first one will already throw